tests/test_async_fetch.py

```diff
@@ -732,35 +730,39 @@ def __int__(self):
                 'select <decimal>$arg',
                 arg="10.2")
 
-    async def test_async_wait_cancel_01(self):
-        # Test that client protocol handles waits interrupted
-        # by closing.
-        lock_key = tb.gen_lock_key()
-
-        con2 = await self.connect(database=self.con.dbname)
-
-        await self.con.query_one(
-            'select sys::advisory_lock(<int64>$0)',
-            lock_key)
-
-        try:
-            async with tg.TaskGroup() as g:
-
-                async def exec_to_fail():
-                    with self.assertRaises(ConnectionAbortedError):
-                        await con2.query(
-                            'select sys::advisory_lock(<int64>$0)', lock_key)
-
-                g.create_task(exec_to_fail())
-
-                await asyncio.sleep(0.1)
-                await con2.aclose()
-
-        finally:
-            self.assertEqual(
-                await self.con.query(
-                    'select sys::advisory_unlock(<int64>$0)', lock_key),
-                [True])
+    # Advisory locks don't work/ for now
+    # Can't xfail because:
+    #     database "asyncfetch" is being accessed by other users
```

Comment:
Not sure I follow the "database "asyncfetch" is being accessed by other users" -- just curious why you can't xfail.

Reply:
Probably connection objects is kept somewhere in traceback object while we are trying to drop database in `finally`.